src/Storage/Mapping/Type/CarbonDateType.php

```diff
@@ -2,25 +2,11 @@
 
 namespace Bolt\Storage\Mapping\Type;
 
-use Carbon\Carbon;
-use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Types\DateType;
+use Bolt\Common\Deprecated;
+use Bolt\Storage\Database\Schema\Types;
 
-class CarbonDateType extends DateType
-{
-    /**
-     * {@inheritdoc}
-     */
-    public function convertToPHPValue($value, AbstractPlatform $platform)
-    {
-        if ($value === null) {
-            return $value;
-        }
+Deprecated::cls(CarbonDateType::class, 3.3);
 
-        if (empty($value)) {
-            return null;
-        }
-
-        return Carbon::instance(parent::convertToPHPValue($value, $platform));
-    }
-}
+// Load CarbonDateType.php which defines class alias
+class_exists(Types\CarbonDateType::class);
```

Comment:
The class exists call and comment are not needed and wrong. The alias is defined here not in the new class.

Reply:
I just hadn't pushed the update from our conversation, I was still waiting on an answer in Slack :disappointed: